js/source/worker_tile.js

```diff
@@ -232,6 +232,55 @@ WorkerTile.prototype.parse = function(data, layerFamilies, actor, rawTileData, c
     }
 };
 
+// Proof of concept for updating only the properties (reusing existing,
+// already-parsed geometries) of a vector tile.
+//
+// PLEASE NOTE: In practice, this might make more sense as part of a separate,
+// custom source type, rather than being baked into WorkerTile.  It's actually
+// pretty independent of the rest of the code here, except that it reuses
+// isBucketEmpty and serializeBucket.
+//
+// Params analogous to WorkerTile#parse, except that `data`, rather than being
+// a `VectorTile`, is an object like:
+// {
+//     source-layer-name: [{ /* feature 0 properties */}, {/* feature 1 properties */}, ...],
+//     another-layer-name: [...]
+// }
+// where feature 0, 1, 2 are the features, in order, in this vector tile.
```

Comment:
Should this comment be reformatted into JSDoc like rest of the code, while moving the note to a ticket or GH comment? It looks a bit like a TODO.